Parsers/ASimAuthentication/Parsers/ASimAuthenticationMicrosoftWindowsEvent.yaml

```diff
@@ -15,6 +15,7 @@ References:
 Description: |
   This ASIM parser supports normalizing Windows Authentication events (4624, 4625, 4634, and 4647), collected either by the Log Analytics Agent or the Azure Monitor Agent, into either the WindowsEvent (WEF) or SecurityEvent tables, to the ASIM Authentication schema.
 ParserName: ASimAuthenticationMicrosoftWindowsEvent
+EquivalentBuiltInParser: _ASim_Authentication_WindowsEvent
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Sorted columns